rls-rustc/src/lib.rs

```diff
@@ -19,7 +19,7 @@ use std::env;
 use std::process;
 
 pub fn run() {
-    drop(env_logger::init());
+    env_logger::init();
```

Comment:
This was weird.
I don't understand how the drop() call would do anything useful.

Reply:
I imagine it was supposed to be a "shorthand" for `let _ = env_logger::try_init();`? Not sure :man_shrugging: